examples/advanced_stream.rs

```diff
@@ -0,0 +1,46 @@
+use std::env;
```

Comment:
It is more advanced because it uses a lot of new features for streaming that were not previously present in the SDK.

We could just update the simple_stream.rs and rename it tho